website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,8 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## June 2024
 
+- **Enhancement:** Updates to the UI when [creating merge jobs](/docs/deploy/merge-jobs) are now available. The updates include improvements to helper text, new deferral settings, and more. 
```

Comment:
Since it's hard to tell from the linked page everything new, "and more" might leave folks guessing. If and more is like behind-the-scenes, maybe we could say
```suggestion
- **Enhancement:** Updates to the UI when [creating merge jobs](/docs/deploy/merge-jobs) are now available. The updates include improvements to helper text, new deferral settings, and performance improvements. 
```

Otherwise we should probably list everything new here since it's a release note.

Reply:
like it!